Task3/Task3_Atanas_Kisyov/README.md

```diff
@@ -0,0 +1,13 @@
+# Connect to PostgreSQL with Python
+
+## View Table
```

Comment:
Please think about name of this point.  We will execute script that will create a read-only pool and run SELECT queries.